Form/Type/DatePickerType.php

```diff
@@ -18,7 +22,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add(
                 'date_example1',
-                'date',
```

Comment:
This is deprecated and will be removed in 3.0, see https://github.com/symfony/symfony/blob/master/UPGRADE-3.0.md#form